DNN Platform/DotNetNuke.Abstractions/Portals/IPortalSettings.cs

```diff
@@ -220,6 +220,7 @@ public interface IPortalSettings
         /// Gets a value indicating whether gets whether the Skin Widgets are enabled/supported.
         /// </summary>
         /// <remarks>Defaults to True.</remarks>
+        [Obsolete("This setting is no longer relevant as skin widgets are no longer supported. Scheduled for removal in DNN 11.")]

```

Comment:
```suggestion
        [Obsolete("This setting is no longer relevant as skin widgets are no longer supported. Scheduled for removal in v11.0.0")]

```
THis way we match the format of others so we don't miss this later.